spark_log_parser/parsing_models/executor_model.py

```diff
@@ -1,15 +1,11 @@
-import numpy
-
-
 class ExecutorModel:
     """
-    Model for a task within a stage
+    Model for a Spark Executor (i.e. worker node)
```

Comment:
Since we have our `ExecutorModel`s stored in a `defaultdict` now, we need these classes to have no-arg constructors. We will be populating these fields manually anyway on these models as we receive new log lines.